hedera-mirror-rest/api/v1/openapi.yml

```
@@ -923,6 +952,54 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Contract'
+    ContractResult:
+      type: object
+      properties:
+        amount:
+          description: The
+          example: 10
+          type: number
```

Number includes floating point
```suggestion
          type: int64
```

---

Updated